scripts/globals/magic_utils/spell_damage.lua

```diff
@@ -213,11 +213,21 @@ xi.magic_utils.spell_damage.calculateSDT = function(caster, target, spell, spell
         SDTMod = target:getMod(xi.magic.specificDmgTakenMod[spellElement])
 
         -- TODO: Ths conversion right here needs to be updated once work on SDT modifiers is finished.
+        -- Current behaviour goes like this:
+        -- In the database, the modifiers as set as the numbers we actually want.
```

Comment:
do we tho? do we? well, you want that number _here_ but what we're doing there is converting a % into a modifier because modifiers are integers and the sql field currently isn't

Reply:
I could be more clear, for sure. I meant that, right now, the database holds the values as we want to use them here. Not that we want the database values to be that way.

Reply:
yeah but we want them to understand we can't simply pass `0.250` all the way across or spend our time explaining what we meant when they read it

Reply:
I've edited it to make it a lil more clear, and our future plans to change the database values.